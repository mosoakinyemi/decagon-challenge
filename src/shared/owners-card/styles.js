import {StyleSheet} from 'react-native';
import {colors} from '../../colors';
import {fonts} from '../../fonts';
import {hp, wp} from '../resposive-dimension';

export const styles = StyleSheet.create({
  icon: {
    width: wp(20),
  },
  bioDetail: {
    ...fonts.regularText14,
  },
  bio: {},
  containerRow: {
    // flexDirection: 'row',
    paddingHorizontal: wp(10),
  },
  name: {
    ...fonts.boldText18,
    backgroundColor: colors.ash,
    minHeight: hp(30),
    paddingLeft: hp(10),
  },
  row: {
    flexDirection: 'row',
    borderBottomWidth: 0.5,
    borderBottomColor: colors.ash,
    alignItems: 'center',
  },

  detail: {
    width: wp(130),
    ...fonts.regularText14,
  },
  listTitle: {
    width: wp(20),
    height: wp(20),
    ...fonts.boldText16,
    backgroundColor: colors.gray,
    marginRight: wp(5),
    marginVertical: wp(2),
  },
  card: {
    alignSelf: 'center',
    width: wp(350),
    borderRadius: wp(5),
    paddingBottom: hp(15),
    backgroundColor: colors.white,
    overflow: 'hidden',

    shadowColor: '#000',
    shadowOffset: {
      width: 0,
      height: 2,
    },
    shadowOpacity: 0.25,
    shadowRadius: 3.84,

    elevation: 5,
    marginVertical: hp(5),
  },
});
