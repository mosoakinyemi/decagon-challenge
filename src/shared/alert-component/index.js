import React, {useState} from 'react';
import {View, Text, Modal} from 'react-native';
import {styles} from './styles';

export const AlertComponent = ({text}) => {
  return (
    // <Modal animationType="slide" transparent={true} visible={true}>
    <View style={styles.container}>
      <Text style={styles.text}>{text}</Text>
    </View>
    // </Modal>
  );
};
