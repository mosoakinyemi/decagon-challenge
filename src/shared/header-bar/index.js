import React from 'react';
import {View, Text, TouchableOpacity} from 'react-native';
import {styles} from './styles';
import Icon from 'react-native-vector-icons/Entypo';
import {hp} from '../resposive-dimension';

const HeaderBar = ({title, onRightPress, onLeftPress}) => {
  return (
    <View style={styles.container}>
      {onLeftPress && (
        <TouchableOpacity style={styles.sideIcon} onPress={onLeftPress}>
          <Icon name="chevron-thin-left" size={hp(16)} />
          <Text style={styles.backText}>Back</Text>
        </TouchableOpacity>
      )}
      <Text style={styles.title}>{title}</Text>
      <TouchableOpacity style={styles.sideIcon} onPress={onRightPress}>
        {onRightPress && <Text style={styles.backText}>Home</Text>}
      </TouchableOpacity>
    </View>
  );
};

export default HeaderBar;
