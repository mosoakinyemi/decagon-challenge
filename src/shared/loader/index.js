import React from 'react';
import {ActivityIndicator, View, Text} from 'react-native';
import {colors} from '../../colors';
import {styles} from './styles';

export const Loader = ({text}) => {
  return (
    <View style={styles.loader}>
      <ActivityIndicator size="large" color={colors.dark} />
      <Text style={styles.text}>{text}</Text>
    </View>
  );
};
