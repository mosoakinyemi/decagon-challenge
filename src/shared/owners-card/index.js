import React, {PureComponent} from 'react';
import {View, Text} from 'react-native';
import {styles} from './styles';
import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
import {wp} from '../resposive-dimension';

export class OwnersCard extends PureComponent {
  render() {
    const {
      bio,
      car_color,
      car_model,
      car_model_year,
      gender,
      job_title,
      email,
      first_name,
      last_name,
    } = this.props.data;
    return (
      <View style={styles.card}>
        <Text style={styles.name}>
          {first_name} {last_name} ({gender})
        </Text>

        <View style={styles.containerRow}>
          <View>
            <View style={styles.row}>
              <View style={styles.row}>
                <MaterialIcons name="work" size={wp(15)} style={styles.icon} />
                <Text style={styles.detail}>{job_title}</Text>
              </View>

              <MaterialIcons name="email" size={wp(15)} style={styles.icon} />
              <Text style={styles.detail}>{email}</Text>
            </View>

            <View style={styles.row}>
              <View style={styles.row}>
                <MaterialIcons
                  name="directions-car"
                  size={wp(15)}
                  style={styles.icon}
                />
                <Text style={styles.detail}>
                  {car_model} {car_model_year}
                </Text>
              </View>
              <MaterialIcons
                name="color-lens"
                size={wp(15)}
                style={styles.icon}
              />
              <Text style={styles.detail}>{car_color}</Text>
            </View>
          </View>

          <View style={styles.bioSection}>
            <Text style={styles.bio}>Bio:</Text>
            <Text style={styles.bioDetail}>{bio}</Text>
          </View>
        </View>
      </View>
    );
  }
}
