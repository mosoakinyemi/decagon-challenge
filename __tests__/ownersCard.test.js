import React from 'react';
import renderer from 'react-test-renderer';
import {OwnersCard} from '../src/shared/owners-card';

jest.useFakeTimers();

describe('<OwnersCard/>', () => {
  it('OwnersCard component renders correctly', () => {
    mockData.map((data) => {
      const instance = renderer.create(<OwnersCard data={data} />);
      expect(instance).toMatchSnapshot();
    });
  });
});

const mockData = [
  {
    bio: 'lorem ipsum ndhd fdjdjjd d dsssns',
    car_color: 'red',
    car_model: 'Toyota',
    car_model_year: '2008',
    gende: 'male',
    job_title: 'Senior Developer',
    email: 'rajibola1997@gmail.com',
    first_name: 'ridwan',
    last_name: 'ajibola',
  },
  {
    bio: 'lorem ipsum ndhd fdjdjjd d dsssns',
    car_color: 'blue',
    car_model: 'Mazda',
    car_model_year: '2004',
    gende: 'female',
    job_title: 'Junior Developer',
    email: 'rajibola1997@gmail.com',
    first_name: 'john',
    last_name: 'doe',
  },
  {
    bio: 'lorem ipsum ndhd fdjdjjd d dsssns',
    car_color: 'green',
    car_model: 'Toyota',
    car_model_year: '2008',
    gende: 'male',
    job_title: 'DevOp Developer',
    email: 'rajibola1997@gmail.com',
    first_name: 'shishi',
    last_name: 'ma',
  },
  {
    bio: 'lorem ipsum ndhd lured d disius',
    car_color: 'brown',
    car_model: 'BMW',
    car_model_year: '2008',
    gende: 'male',
    job_title: 'Senior Developer',
    email: 'rajibola1997@gmail.com',
    first_name: 'kaka',
    last_name: 'demo',
  },
];
