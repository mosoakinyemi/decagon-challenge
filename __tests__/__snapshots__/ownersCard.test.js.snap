// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OwnersCard/> OwnersCard component renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#fff",
      "borderRadius": 10,
      "elevation": 3,
      "marginVertical": 8,
      "overflow": "hidden",
      "paddingBottom": 24.5,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
      "width": 700,
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "#f4f4f8",
        "color": "#2f3138",
        "fontSize": 29.5,
        "fontWeight": "bold",
        "lineHeight": 44.5,
        "minHeight": 49.5,
        "paddingLeft": 16.5,
      }
    }
  >
    ridwan
     
    ajibola
     (
    )
  </Text>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Senior Developer
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          rajibola1997@gmail.com
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Toyota
             
            2008
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          red
        </Text>
      </View>
    </View>
    <View>
      <Text
        style={Object {}}
      >
        Bio:
      </Text>
      <Text
        style={
          Object {
            "color": "#2f3138",
            "fontSize": 23,
            "lineHeight": 29.5,
          }
        }
      >
        lorem ipsum ndhd fdjdjjd d dsssns
      </Text>
    </View>
  </View>
</View>
`;

exports[`<OwnersCard/> OwnersCard component renders correctly 2`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#fff",
      "borderRadius": 10,
      "elevation": 3,
      "marginVertical": 8,
      "overflow": "hidden",
      "paddingBottom": 24.5,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
      "width": 700,
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "#f4f4f8",
        "color": "#2f3138",
        "fontSize": 29.5,
        "fontWeight": "bold",
        "lineHeight": 44.5,
        "minHeight": 49.5,
        "paddingLeft": 16.5,
      }
    }
  >
    john
     
    doe
     (
    )
  </Text>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Junior Developer
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          rajibola1997@gmail.com
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Mazda
             
            2004
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          blue
        </Text>
      </View>
    </View>
    <View>
      <Text
        style={Object {}}
      >
        Bio:
      </Text>
      <Text
        style={
          Object {
            "color": "#2f3138",
            "fontSize": 23,
            "lineHeight": 29.5,
          }
        }
      >
        lorem ipsum ndhd fdjdjjd d dsssns
      </Text>
    </View>
  </View>
</View>
`;

exports[`<OwnersCard/> OwnersCard component renders correctly 3`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#fff",
      "borderRadius": 10,
      "elevation": 3,
      "marginVertical": 8,
      "overflow": "hidden",
      "paddingBottom": 24.5,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
      "width": 700,
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "#f4f4f8",
        "color": "#2f3138",
        "fontSize": 29.5,
        "fontWeight": "bold",
        "lineHeight": 44.5,
        "minHeight": 49.5,
        "paddingLeft": 16.5,
      }
    }
  >
    shishi
     
    ma
     (
    )
  </Text>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            DevOp Developer
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          rajibola1997@gmail.com
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Toyota
             
            2008
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          green
        </Text>
      </View>
    </View>
    <View>
      <Text
        style={Object {}}
      >
        Bio:
      </Text>
      <Text
        style={
          Object {
            "color": "#2f3138",
            "fontSize": 23,
            "lineHeight": 29.5,
          }
        }
      >
        lorem ipsum ndhd fdjdjjd d dsssns
      </Text>
    </View>
  </View>
</View>
`;

exports[`<OwnersCard/> OwnersCard component renders correctly 4`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#fff",
      "borderRadius": 10,
      "elevation": 3,
      "marginVertical": 8,
      "overflow": "hidden",
      "paddingBottom": 24.5,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
      "width": 700,
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "#f4f4f8",
        "color": "#2f3138",
        "fontSize": 29.5,
        "fontWeight": "bold",
        "lineHeight": 44.5,
        "minHeight": 49.5,
        "paddingLeft": 16.5,
      }
    }
  >
    kaka
     
    demo
     (
    )
  </Text>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            Senior Developer
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          rajibola1997@gmail.com
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#f4f4f8",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#f4f4f8",
              "borderBottomWidth": 0.5,
              "flexDirection": "row",
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 30,
                },
                Object {
                  "width": 40,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
          <Text
            style={
              Object {
                "color": "#2f3138",
                "fontSize": 23,
                "lineHeight": 29.5,
                "width": 260,
              }
            }
          >
            BMW
             
            2008
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              Object {
                "width": 40,
              },
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
        <Text
          style={
            Object {
              "color": "#2f3138",
              "fontSize": 23,
              "lineHeight": 29.5,
              "width": 260,
            }
          }
        >
          brown
        </Text>
      </View>
    </View>
    <View>
      <Text
        style={Object {}}
      >
        Bio:
      </Text>
      <Text
        style={
          Object {
            "color": "#2f3138",
            "fontSize": 23,
            "lineHeight": 29.5,
          }
        }
      >
        lorem ipsum ndhd lured d disius
      </Text>
    </View>
  </View>
</View>
`;
